microsoft-edge/webview2/how-to/set-preview-channel.md

```diff
@@ -0,0 +1,290 @@
+
+1.  Expand **Local Computer Policy**, then **Computer Configuration** or **User Configuration**.  Then expand **Administrative Templates** > **Microsoft Edge WebView2**.
+
+    :::image type="complex" source="./media/local-group-policy-editor.png" alt-text="Local Group Policy Editor dialog box" lightbox="./media/local-group-policy-editor.png":::
+       **Local Group Policy Editor** dialog box
+    :::image-end:::  
+
+    <!-- where to put this? -->
+    The registry location is: `{HKLM/HKCU}\Software\Policies\Microsoft\Edge\WebView2\{override name}`
+
+1.  Select **Browser Executable Folder** and then fill-in the **Browser Executable Folder** dialog box.  The following screenshots apply to setting the Browser Executable Folder.  Alternatively, select **Release Channel Preference**, which uses similar dialog boxes.
+
+    :::image type="complex" source="./media/browser-executable-folder.png" alt-text="Setting the Browser Executable Folder" lightbox="./media/browser-executable-folder.png":::
+       Setting the **Browser Executable Folder**
+    :::image-end:::  
+
+1.  Select the **Show** button.<!--Select the **Next Setting** button. ?-->
+
+1.  Fill-in the **Show Contents** dialog box.
+
+    :::image type="complex" source="./media/show-contents.png" alt-text="The Show Contents dialog box" lightbox="./media/show-contents.png":::
+       The **Show Contents** dialog box
+    :::image-end:::  
+
+1.  Select **OK** to close the dialog boxes.
+
+For more information, see [Configure Microsoft Edge policy settings](/deployedge/configure-microsoft-edge).
+
+
+<!-- 3. Registry Override ==================================================-->
+## Using a registry override
+<!-- ## Specifying a channel by using a registry override -->
+
+When specifying a preview channel by using a registry override, there are two options:
+*  Change the browser executable folder.
+*  Change the release channel preference.
+
+These two approaches are described below.
+
+<!-- 3a. registry override: browser executable folder -->
+### By setting the browser executable folder
+
+To make your application use a Microsoft Edge preview channel by using a registry override that sets the browser executable folder:
+
+1.  Open a Powershell command prompt.<!-- delete "Powershell"? the article elsewhere mentions "cmd.exe" -->
+
+1.  Modify and then run the following command:
+
+    `REG ADD HKLM\Software\Policies\Microsoft\Edge\WebView2\BrowserExecutableFolder /v * /t REG_SZ /d "C:\Users\myname\AppData\Local\Microsoft\Edge SxS\Application\88.0.680.0"`
+
+    The asterisk (*) as the value name makes this override apply to all WebView2 apps.  If you want to only apply this override to a particular WebView2 app, replace the asterisk by the filename of the app's executable.
+
+    Replace `C:\Users\myname\AppData\Local\Microsoft\Edge SxS\Application\88.0.680.0` by the path to the desired Microsoft Edge preview channel.
+
+#### Resuming using the default, WebView2 Evergreen Runtime
```

Comment:
@jasonstephen15
Possible missing info: The bottom of each of the 4 or 5 sections might need to say how to switch back to using the Runtime instead of a preview channel.
Do the other approaches have enough info to help the dev switch back to using the Runtime instead of a preview channel?
1) code
2) group policy
4) environment variable
Will the dev be stumped when they are done using a preview channel and want to switch back to using the Runtime?
Rendered article: 
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/how-to/set-preview-channel?branch=pr-en-us-1333#resuming-using-the-default-webview2-evergreen-runtime

Reply:
Yes - I think that the average dev would know how to do that.